rq/worker.py

```diff
@@ -139,7 +139,8 @@ def __init__(
         connection: Optional['Redis'] = None,
         exc_handler=None,
         exception_handlers=None,
-        default_worker_ttl=DEFAULT_WORKER_TTL,
+        default_worker_ttl=DEFAULT_WORKER_TTL,  # TODO remove this arg in 2.0
+        worker_ttl=DEFAULT_WORKER_TTL,
```

Comment:
`worker_ttl` should be optional. If `worker_ttl` is `None` and `default_worker_ttl` is supplied, `default_worker_ttl` should still be used in line 157.

Reply:
I'm quite confused about what you would like the end result to be here. 

The original ask in the linked ticket was:

> Add a deprecation warning to default_worker_ttl and change it to worker_ttl, I'd like this to be released as part of RQ v2.0

I changed `default_worker_ttl` -> `worker_ttl`  like you asked.

> worker_ttl should be optional.

It is currently optional since it has a default value of `DEFAULT_WORKER_TTL`

>  If worker_ttl is None and default_worker_ttl is supplied, default_worker_ttl should still be used in line 157.

Why have both `worker_ttl` and `default_worker_ttl` parameters? In the linked ticket it was suggested to rename it.

Also the renaming aligns it with the other parameters, i.e. they don't have `default` in the name.

For example:

`maintenance_interval: int = DEFAULT_MAINTENANCE_TASK_INTERVAL,` and `job_monitoring_interval=DEFAULT_JOB_MONITORING_INTERVAL,`

---
 
Let me know if I'm misunderstanding something but I'm having a hard time understanding what else there is to be done here.

Thanks

Reply:
It should be something like this:
```python
default_worker_ttl=DEFAULT_WORKER_TTL,  # TODO remove this arg in 2.0
worker_ttl=None,
```

```python
if worker_ttl:
    worker_ttl = worker_ttl
elif default_worker_ttl:
    worker_ttl = default_worker_ttl
else:
    worker_ttl = DEFAULT_WORKER_TTL
```

I'd like it so that people using `Worker(default_worker_ttl=180)` to still work while it's being deprecated.